src/github.com/matrix-org/dendrite/syncapi/consumers/roomserver.go

```diff
@@ -28,15 +28,15 @@ import (
 	sarama "gopkg.in/Shopify/sarama.v1"
 )
 
-// Server contains all the logic for running a sync server
-type Server struct {
+// Roomserver consumes events that originated in the room server.
+type Roomserver struct {
```

Comment:
Could this be called `Consumer` or something along those lines to make it clearer that it consumes things?